config/supported_ssg.default.yaml

```diff
@@ -1,323 +1,27 @@
 ---
```

Comment:
Why did you edit this file?

Reply:
Your tests should not touch it either, if paths are properly mocked.

Reply:
on accident